content/en/docs/examples/virtual-machines/bookinfo/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Bookinfo with a Virtual Machine
-description: Run the Bookinfo application with a MySQL service running on a virtual
+title: Examples of using virtual machine integration with Istio
```

Comment:
My understanding was that these tutorials were aimed at adding non-k8s services to a mesh. Maybe we should title it as such?

Reply:
Adding external services to the mesh, perhaps?

As an Istio user, I am much more likely use that as a search query and stumble upon these docs.